src/cdk/testing/private/e2e/asserts.ts

```diff
@@ -40,7 +40,7 @@ export async function expectLocation(element: FinderResult, {x, y}: Point) {
  * Asserts that one element is aligned with another.
  */
 export async function expectAlignedWith(element: FinderResult, otherElement: FinderResult) {
-  await getElement(otherElement).getLocation().then((location: Point) => {
-    expectLocation(getElement(element), location);
+  await getElement(otherElement).getLocation().then(async (location: Point) => {
+    await expectLocation(getElement(element), location);
```

Comment:
For some reason there was a missing await here that caused e2e tests to fail. Not sure if it was flaky because it happened constantly on this PR, but the missing `await` definitely is a reasonable for flakiness..